packages/inventory-compliance/src/PresentationalComponents/ConditionalLink.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 const ConditionalLink = ({ children, href, ...additionalProps }) => (
-    href && <a href={ href } { ...additionalProps }>{ children }</a> || children
+    href && <a href={ href } { ...additionalProps }>{ children }</a> || children || ''
```

Comment:
I noticed that if you have some references/identifiers in the compliance db that are returning nil, this component isn't able to render - and causes the whole page to fail. This is just a side-note and should hardly be called (it just returns an empty string if needed)